connect/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@gala-chain/connect",
+  "version": "0.0.1",
```

Comment:
I assume the version will be aligned to what we currently have (1.1.17)
@dzikowski wdyt?

Reply:
Yes, and we should add it to ensure versions script